src/lib/containers/drug_upload_tool/Header.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+
+export type HeaderProps = {
+  copy?: string;
```

Comment:
It looks like the variable `copy` is an html string, it seems like `html` would be a more fitting name?